src/yang/ietf-bgp.yang

```diff
@@ -689,23 +689,37 @@ module ietf-bgp {
             }
             container messages {
               description
-                "Counters for BGP messages sent and received from the
+		"Counters for BGP messages sent and received from the
                  neighbor";
-              leaf in-total-messages {
-                type yang:zero-based-counter32;
+              leaf messages-received {
+                type yang:zero-based-counter64;
                 description
-                  "The total number of messages received
-                   from the remote peer on this connection.";
+		  "Number of BGP messages received from this neighbor";
                 reference
-                  "RFC 4271, Section 4.";
+                  "RFC 4273 - bgpPeerInTotalMessages";
```

Comment:
Thanks for catching the wrong RFC number. Just a nit. I have tried to follow the format of

`RFC AAAA: [Title of the RFC], <Section number of the RFC>.`

where [] is required and `<>` is optional to have.

Reply:
Updated in next diff.